src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -608,6 +609,19 @@ public void reloadSession(String userName, String botName) throws IOException {
     }
   }
 
+  /**
+   * Get all current predicates names and their values
+   * for the current session
+   * @return
+   */
+  public Map<String, String> getPredicates() {

```

Comment:
there should be a getPredicates(session) method.. and this method should call that with the default session.

Reply:
agreed fixed